diff --git a/app/browser/Showcase.ts b/app/browser/Showcase.ts
index 3038781..5156bcf 100644
--- a/app/browser/Showcase.ts
+++ b/app/browser/Showcase.ts
@@ -5,7 +5,9 @@ import { SHOWCASE_CONTROL, SHOWCASE_COPY } from './constants.js'
 import { AlertSection } from './sections/AlertSection.js'
 import { AccordionSection } from './sections/AccordionSection.js'
 import { ButtonGroupSection } from './sections/ButtonGroupSection.js'
+import { BackgroundSection } from './sections/BackgroundSection.js'
 import { BadgeSection } from './sections/BadgeSection.js'
+import { BorderSection } from './sections/BorderSection.js'
 import { BreadcrumbSection } from './sections/BreadcrumbSection.js'
 import { ButtonSection } from './sections/ButtonSection.js'
 import { CardSection } from './sections/CardSection.js'
@@ -137,6 +139,8 @@ export class Showcase implements ShowcaseInterface {
 			new PositionSection(this.#main),
 			new SizingSection(this.#main),
 			new VisibilitySection(this.#main),
+			new BackgroundSection(this.#main),
+			new BorderSection(this.#main),
 			new NavbarSection(this.#main),
 		]
 	}
