diff --git a/app/browser/index.ts b/app/browser/index.ts
index 474aa0a..fbacfce 100644
--- a/app/browser/index.ts
+++ b/app/browser/index.ts
@@ -38,4 +38,6 @@ export * from './sections/FlexSection.js'
 export * from './sections/PositionSection.js'
 export * from './sections/SizingSection.js'
 export * from './sections/VisibilitySection.js'
+export * from './sections/BackgroundSection.js'
+export * from './sections/BorderSection.js'
 export * from './sections/NavbarSection.js'
